app/components/award_details/view.rb

```diff
@@ -13,5 +13,9 @@ def initialize(trainee)
     def award_date
       date_for_summary_view(trainee.awarded_at)
     end
+
+    def show_remove_qts_link?
+      trainee.awarded?
+    end
```

Comment:
this may need a check - either in here, or in the actual view - to only show the link if the user is an admin